packages/workspace/src/schematics/new/new.ts

```diff
@@ -309,25 +310,16 @@ function setESLintDefault() {
  */
 function setTSLintDefault() {
   return updateWorkspaceInTree((json) => {
-    if (!json.schematics) {
-      json.schematics = {};
-    }
-    json.schematics['@nrwl/workspace'] = { library: { linter: 'tslint' } };
-    json.schematics['@nrwl/cypress'] = {
-      'cypress-project': { linter: 'tslint' },
-    };
-    json.schematics['@nrwl/node'] = {
-      application: { linter: 'tslint' },
-      library: { linter: 'tslint' },
-    };
-    json.schematics['@nrwl/nest'] = {
-      application: { linter: 'tslint' },
-      library: { linter: 'tslint' },
-    };
-    json.schematics['@nrwl/express'] = {
-      application: { linter: 'tslint' },
-      library: { linter: 'tslint' },
-    };
+    setDefault(json, '@nrwl/workspace', 'library', 'linter', 'tslint');
```

Comment:
This looks good @vsavkin. Can you please update it [here](https://github.com/mehrad-rafigh/nx/blob/93bce221b52df1ebebd29c4416983da82228d0db/packages/angular/src/schematics/init/init.ts#L124) as well?

Reply:
We use a different api in there, so won't work as is. We are going to rework that in January, so I'll update it then